src/main/resources/lang/en-GB.json

```diff
@@ -100,6 +100,17 @@
       "guild_member_count": "Member Count",
       "guild_id": "ID: {0}"
     },
+    "slowmode": {
+      "command": "slowmode",
+      "description": "Sets the slowmode of a channel",
+      "time_exceeded": "Maximum time for slowmode of channels is 6 hours",
+      "invalid_channel" : "I cannot find channel `{0}`",
```

Comment:
This should already be under responses.

Reply:
https://github.com/CascadeBot/CascadeBot/pull/239/files#diff-f7944c7ae4f73ab92f3d2297e07d5aa9R1261